hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/processor/SinkProcessors.java

```diff
@@ -373,12 +374,14 @@ public static ProcessorMetaSupplier writeRemoteListP(@Nonnull String listName, @
             @Nonnull String updateQuery,
             @Nonnull SupplierEx<? extends CommonDataSource> dataSourceSupplier,
             @Nonnull BiConsumerEx<? super PreparedStatement, ? super T> bindFn,
+            int batchLimit,
```

Comment:
This is a breaking change in Core API. While we can do that in minor versions, we should at least guide the user what to use for the new parameter. We should point him to the `DEFAULT_BATCH_LIMIT` constant. We should also add the new parameter to the end.

Reply:
Thanks for the suggestions.
How do you propose to direct the user to the new constant? Explanation in the release notes?

> We should also add the new parameter to the end.

Ah sorry about that, not sure why I added it there, maybe lack of coffee :) Will update.

Reply:
In javadoc it's fine, a `@param` tag.

Reply:
I added an explanation to the javadoc as a second paragraph instead of a `@param` tag. I think it looks better:
```
     /**
      * Returns a supplier of processors for {@link Sinks#jdbcBuilder()}.
      * <p>
      * Until Jet 4.4, the batch size limit for the supplied processors was
      * fixed to 50. Since Jet 4.5, this limit is now freely configurable
      * using the {@code batchLimit} parameter.
      */
```

Let me know if you still prefer a `@param` annotation.

Reply:
I just got thinking that an alternative to avoid the breaking change entirely could be to implement an additional overloaded method without the `batchLimit` parameter (like it was previously) that simply calls the new method with the default for `batchLimit`.

What do you think?